README.md

```diff
@@ -112,18 +110,78 @@ nix-shell
+
+```sh
+$ hoprd --help
+Options:
+  --help                        Show help  [boolean]
+  --version                     Show version number  [boolean]
+  --network                     Which network to run the HOPR node on  [choices: "ETHEREUM"] [default: "ETHEREUM"]
+  --provider                    A provider url for the Network you specified  [default: "wss://still-patient-forest.xdai.quiknode.pro/f0cdbd6455c0b3aea8512fc9e7d161c1c0abf66a/"]
+  --host                        The network host to run the HOPR node on.  [default: "0.0.0.0:9091"]
+  --announce                    Announce public IP to the network  [boolean] [default: false]
+  --admin                       Run an admin interface on localhost:3000  [boolean] [default: false]
+  --rest                        Run a rest interface on localhost:3001  [boolean] [default: false]
+  --restHost                    Updates the host for the rest server  [default: "localhost"]
+  --restPort                    Updates the port for the rest server  [default: 3001]
+  --healthCheck                 Run a health check end point on localhost:8080  [boolean] [default: false]
+  --healthCheckHost             Updates the host for the healthcheck server  [default: "localhost"]
+  --healthCheckPort             Updates the port for the healthcheck server  [default: 8080]
+  --password                    A password to encrypt your keys  [default: ""]
+  --identity                    The path to the identity file  [default: "/root/.hopr-identity"]
+  --run                         Run a single hopr command, same syntax as in hopr-admin  [default: ""]
+  --dryRun                      List all the options used to run the HOPR node, but quit instead of starting  [boolean] [default: false]
+  --data                        manually specify the database directory to use  [default: ""]
+  --init                        initialize a database if it doesn't already exist  [boolean] [default: false]
+  --adminHost                   Host to listen to for admin console  [default: "localhost"]
+  --adminPort                   Port to listen to for admin console  [default: 3000]
+  --testAnnounceLocalAddresses  For testing local testnets. Announce local addresses.  [boolean] [default: false]
+  --testPreferLocalAddresses    For testing local testnets. Prefer local peers to remote.  [boolean] [default: false]
+```
+
+As you might have noticed running the node without any command-line argument might not work depending on the installation method used. Here are examples to run a node with some safe configurations set.
+
+### Using NPM
+
+The following command assumes you've setup an alias like described in [Install via NPM](#install-via-npm).
+
+```sh
+hoprd --identity ${HOME}/.hopr-identity --password switzerland --init --announce --admin
+```
+
+Here is a short break-down of each argument.
+
+```sh
+hoprd
```

Comment:
Same, I would rather to have this as a separate link to our flags docs. [Here's](https://developers.ceramic.network/reference/typescript/classes/_ceramicnetwork_core.ceramic.html) and example.

Reply:
This is user-facing docs whereas the auto-generated docs are meant for devs.